index.js

```diff
@@ -311,6 +322,7 @@ module.exports = (command, args, options) => {
 	// eslint-disable-next-line promise/prefer-await-to-then
 	spawned.then = (onFulfilled, onRejected) => handlePromise().then(onFulfilled, onRejected);
 	spawned.catch = onRejected => handlePromise().catch(onRejected);
+	spawned.cancel = () => processDone.cancel();
```

Comment:
This could be merged with `processDone.cancel()`. I.e. if `processDone.cancel = function {` was `spawned.cancel = function () {` instead, then this line of code could be removed.

Reply:
You mean it should be like this 
```
spawned.cancel = processDone.cancel;
```

Reply:
No:
  - removing the `spawned.cancel = () => processDone.cancel()'` line
  - replacing `processDone.cancel = function {` by `spawned.cancel = function () {`